modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -49,6 +49,14 @@ $ oc -n openshift-adp create secret generic cloud-credentials \
   --from-file=${SCRATCH}/credentials
 ----
 
+[NOTE]
+====
+In {product-title} versions 4.14 and later, the OADP Operator supports a new standardized STS workflow via the Operator Lifecycle Manager (OLM)
```

Comment:
Does via in this context refer to "routing data" ? https://www.ibm.com/docs/en/ibm-style?topic=word-usage#v

Reply:
@stevsmit 
No, not really. I've changed it to 'through'. Commits are squashed. Thank you!